clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/xml/ClusteringCacheManagerServiceConfigurationParser.java

```diff
@@ -234,37 +232,19 @@ public URI getNamespace() {
    */
   @Override
   public Element unparseServiceCreationConfiguration(final ServiceCreationConfiguration<ClusteringService> serviceCreationConfiguration) {
-    try {
-      validateParametersForTranslationToServiceConfig(serviceCreationConfiguration);
-      Document doc = createDocumentRoot();
-      ClusteringServiceConfiguration clusteringServiceConfiguration = (ClusteringServiceConfiguration)serviceCreationConfiguration;
-      Element rootElement = createConnectionElement(doc, clusteringServiceConfiguration);
-      processTimeUnits(doc, rootElement, clusteringServiceConfiguration);
-      Element serverSideConfigurationElem = processServerSideElements(doc, clusteringServiceConfiguration);
-      rootElement.appendChild(serverSideConfigurationElem);
-      return rootElement;
-    } catch (SAXException | ParserConfigurationException | IOException e) {
-      throw new XmlConfigurationException(e);
-    }
-  }
-
-  private void validateParametersForTranslationToServiceConfig(ServiceCreationConfiguration<ClusteringService> serviceCreationConfiguration) {
-    Objects.requireNonNull(serviceCreationConfiguration, "ServiceCreationConfiguration must not be NULL");
-    if (!(serviceCreationConfiguration instanceof ClusteringServiceConfiguration)) {
-      throw new IllegalArgumentException("Parameter serviceCreationConfiguration must be of type ClusteringServiceConfiguration."
-                                         + "Provided type of parameter is : " + serviceCreationConfiguration.getClass());
-    }
+    Element rootElement = unparseConfig(serviceCreationConfiguration);
+    return rootElement;
   }
 
   private Element createRootUrlElement(Document doc, ClusteringServiceConfiguration clusteringServiceConfiguration) {
-    Element rootElement = doc.createElement(TC_CLUSTERED_NAMESPACE_PREFIX + COLON + CLUSTER_ELEMENT_NAME);
-    rootElement.setAttributeNS(XMLConstants.XMLNS_ATTRIBUTE_NS_URI, "xmlns:" + TC_CLUSTERED_NAMESPACE_PREFIX, getNamespace().toString());
+    Element rootElement = doc.createElementNS(NAMESPACE.toString(), TC_CLUSTERED_NAMESPACE_PREFIX + COLON + CLUSTER_ELEMENT_NAME);
+    rootElement.setAttributeNS(XMLConstants.XMLNS_ATTRIBUTE_NS_URI, XMLNS + TC_CLUSTERED_NAMESPACE_PREFIX, NAMESPACE.toString());
     Element urlElement = createUrlElement(doc, clusteringServiceConfiguration);
     rootElement.appendChild(urlElement);
     return rootElement;
   }
 
-  protected Element createUrlElement(Document doc, ClusteringServiceConfiguration clusteringServiceConfiguration) {
```

Comment:
Do not change the access specifiers of the protected methods in this class as they are used by extensions.